.github/workflows/smoketest.yml

```diff
@@ -0,0 +1,95 @@
+on:
+  pull_request:
```

Comment:
I think we actually want this to be on push, as we are not using merge groups any more.